### DISCUSSION TITLE
PDF Meeting List

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/pdf#72] [Q&A]
Hi everyone, 

We noticed that when we printed the meeting list with Region -> Day that the Regions are mixed throughout. For EX: pg 6 is Atlanta: Atlanta NE: Candler Park. However on page 47or 48 Atlanta: Atlanta NE: Atlanta Chastain appears. What we were hoping to see is all of the Atlanta: Atlanta NE/NW/SE/SW in the same section. Attached is the file for review. I double checked to make sure there were no spaces in the child regions of NE/NW/SE/SW. Am I doing something wrong or missing something?

Thanks in advance for your help and thank as always for the amazing service you all are providing. 
[directory (11).pdf](https://github.com/user-attachments/files/16164530/directory.11.pdf)

**Comment by anchovie91471** [comment]
Hey @ATLCOTech! Thanks for catching this! I'm going to look into this and report back. This issues is related to our PDF microservice so I'm going to move it there.
